app/tests/test_solution_service.py

```diff
@@ -0,0 +1,25 @@
+import unittest
+from app import solution_service
+
+
+class SolutionServiceTests(unittest.TestCase):
+
+    ## The following tests will need to be improved as is_correct
+    ## grows in complexity
+    def test_is_correct_correct(self):
+
+        question = "2 x + 3 = 0"
+        attempt = "x = -\\frac{3}{2}"
+
+        result = solution_service.is_correct(question, attempt, 'mcat', '1')
```

Comment:
I guess this is testing both the LaTeX parsing and the solution solving?
Might be worth investigating if we can split those 2 concerns up at some stage, imagine we don't want to conflate the testing of 2 (likely annoying) bits together

Reply:
true, but don't know if there's much we can do about it, at this stage at least. Explicitly testing sympy's parsing module wouldn't yield much benefit. And whether or not something emerges in the structure of the code this is testing  that means we could test the pure solving logic remains to be seen i guess